phoenicis-tests/src/main/java/org/phoenicis/tests/MockedRepository.java

```diff
@@ -38,17 +38,21 @@
 
     private static class MockRepository implements Repository {
         @Override
-        public List<CategoryDTO> fetchInstallableApplications() {
-            return Collections
-                    .singletonList(
-                            new CategoryDTO.Builder()
-                                    .withApplications(Arrays.asList(new ApplicationDTO.Builder().withName("Engines")
-                                            .withScripts(Arrays.asList(new ScriptDTO.Builder().withScriptName("Wine")
-                                                    .withCompatibleOperatingSystems(Arrays.asList(OperatingSystem.LINUX,
-                                                            OperatingSystem.MACOSX))
-                                                    .withScript(wineScript()).build()))
-                                            .build()))
-                                    .withName("Functions").build());
+        public RepositoryDTO fetchInstallableApplications() {
+            return new RepositoryDTO.Builder()
+                    .withCategories(
+                            Collections.singletonList(
+                                    new CategoryDTO.Builder()
+                                            .withApplications(Arrays.asList(new ApplicationDTO.Builder().withName(
```

Comment:
One advantage of the builder pattern is to avoid building the objects step by steps. 
Code formatting should help us. If it is not the case, we should get rid of it. 
This is not the only place where it causes problems (see Spring configuration)
Agree that it is not related to this PR, but we should track this issue though